app.py

```diff
@@ -19,26 +18,49 @@ def index():
     return {"message": data}
 
 
-
 @app.route('/recommend/popular')
 def popular_movies():
     data = recommend_popular_movies()
     return data
 
 
+@app.route('/html/popular')
+def popular():
+    movies = recommend_popular_movies()
```

Comment:
The indentation of this line is incorrect. It should be indented to align with the 'if' statement on line 25.